projects/packages/stats-admin/src/class-admin-post-list-column.php

```diff
@@ -89,7 +89,30 @@ public function add_stats_post_table_cell( $column, $post_id ) {
 				);
 			} else {
 				// Link to the wp-admin stats page.
-				$stats_post_url = admin_url( sprintf( 'admin.php?page=stats#!/stats/post/%d/%d', $post_id, \Jetpack_Options::get_option( 'id', 0 ) ) );
+				$query_args = array(
+					'from'         => 'postList',
+					'jp_post_type' => get_post_type( $post_id ),
+				);
+
+				$potential_params = array(
```

Comment:
```suggestion
				$list_criteria_params = array(
```
Could we use a more descriptive name for the parameters?